src/SignInForm.js

```diff
@@ -35,7 +35,7 @@ class SignInForm extends Component {
       body: stringify(this.state.fields)
     })
       .then(res => res.status === 400
-        ? res.json().then(this.props.onError('Invalid login.'))
+        ? res.json().then(_ => this.props.onError('Invalid login.'))
```

Comment:
how is this different?

Reply:
Was actually wrong before since `then` takes a function. This passes an anonymous function, rather than return value of `this.prop.onError('Invalid login.')`, which is `undefined`.